### PR TITLE
Issue #13345: Enabled SuppressionXpathSingleFilterExamplesTest

### PR DESCRIPTION
**Pull request by Acesif**
Issue: #13345
Enabled SuppressionXpathSingleFilterExamplesTest.
Replaced its Example.txt with Example.java.
> But here I am facing issues as I am running `mvn clean verify`
> I keep getting the error `Class 'FileX' is public, should be declared in a file named 'FileX.java`
> I am not sure what should I do, any guidance is appreciated

image
Included a screenshot as well if it helps

### CODE REVIEW

**Comment by Acesif** [comment]
@stoyanK7 If you could look into this please? Thanks

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathsinglefilter/Example1.txt [hunk 1]
```diff
@@ -1,28 +0,0 @@
...
-    <module name="SuppressionXpathSingleFilter">
-      <property name="files" value="File(One|Two)\.java"/>
-      <property name="checks" value="MethodName"/>
-      <property name="query" value="(//CLASS_DEF[@text='FileOne']/OBJBLOCK/
-                METHOD_DEF[@text='MyMethod']/IDENT)|
-                (//CLASS_DEF[@text='FileTwo']/OBJBLOCK/METHOD_DEF[@text='MyMethod']/IDENT)"/>
-    </module>
-  </module>
-</module>
-*/
```
Comment (MANISH-K-07) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/filters/suppressionxpathsinglefilter/Example1.txt:13]: Why did we lose the config during transition to .java?
@Acesif , please go through the description again.. You need to understand how our parser works.

**Comment by nrmancuso** [comment]
@Acesif do you plan to continue work here?

**Comment by prathm3** [comment]
@nrmancuso , let me know if I can pick this up.

**Comment by romani** [comment]
@prathm3 , please pick up this work and make it done, thanks a lot !

**Pull request closed without merging**